### ISSUE TITLE
Licence/openness/stealing of ideas

### ISSUE THREAD

**Issue by kdmurray91** [borevitzlab/timestreamlib#45]
Tim & I had a brief discussion on Friday about this all being public. He had some loose fears about being scooped after the upcoming conference & demo, but seemed satisfied that it was public after talking it over.

My thoughts are:

- This is open source under the GPL. If people steal & improve, they MUST publish their code, so we can steal their improvements. Hence open source. 
- Science-wise, to make papers happen, you'll need good genetic resources, the SPC, and a bunch of other shit the average github trawler won't. Thus, no worries from me about this being open from that angle.

In short, I vote we keep everything open, and not worry about this. If any further persuasion is required, I suggest we look across the road for examples of what closing off code does. 

Thoughts?

K
